sdk/certificatemanager/azpki/ci.yml

```diff
@@ -0,0 +1,30 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/template/azpki/
+    - eng/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/template/azpki/
+    - eng/
+
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'template/azpki'
```

Comment:
```suggestion
    ServiceDirectory: 'template/azpki'
    
```

Reply:
I am not sure I understand what the change suggested is here?

Reply:
Lack of terminating newline character I think.

Also, it should be `certificatemanager/azpki` not `template/azpki`.